src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx

```diff
@@ -17,12 +17,52 @@ const ManageSupportUI = () => {
           </div>
         }
       >
+        <span className={styles.warning}>
+          <strong> Warning! </strong>`Agreement` is a parent camp to the list of
+          child camps.If you commit support to Agreement the support of the camp
+          in this list will be removed
+        </span>
+        <Col md={12}>
+          <Tag className={styles.tag_btn}>
+            <div>
+              {""}
+              <span className={styles.count}>{""}</span>
+            </div>
+            <Link href="">
+              <a>1 . Home Page by Code District</a>
+            </Link>
+          </Tag>
+
+          <Tag className={styles.tag_btn}>
+            <div>
+              {""}
+              <span className={styles.count}>{""}</span>
+            </div>
+            <Link href="">
+              <a>2 . Home Page Design Talentelgia</a>
+            </Link>
+          </Tag>
+        </Col>
+        <div className={styles.hrtag}></div>
         <div className={styles.notes}>
           {" "}
           Note : To change support order of camp, drag & drop the camp box on
```

Comment:
Same as before

Reply:
It is a static design implementation for now. will be dynamic once API will be implemented.